eng/Generate.ps1

```diff
@@ -44,14 +44,16 @@ $testNames =
     'lro',
     'media_types',
     'model-flattening',
+    'multiple-inheritance',
     'non-string-enum',
+    'object-type',
     'paging',
     'required-optional',
     'url',
+    'url-multi-collectionFormat',
     'validation',
     'xml-service',
-    'xms-error-responses',
-    'url-multi-collectionFormat';
```

Comment:
Just fixed the ordering here since I forgot to do that in the xms-error-response PR.